net/net.sh

```diff
@@ -948,6 +949,17 @@ while [[ -n $1 ]]; do
     elif [[ $1 = --skip-require-tower ]]; then
       maybeSkipRequireTower="$1"
       shift 1
+    elif [[ $1 = --client-type ]]; then
+      clientType=$2
```

Comment:
You'll need to actually pass this variable to the client, in here: https://github.com/solana-labs/solana/blob/daf71003d30c9e3dac48b4391e03c33025b88c61/net/net.sh#L413